pkg/metrics/metrics.go

```diff
@@ -48,6 +44,16 @@ type PrometheusNodeConfig struct {
 	namespace string
 }
 
+// MetricsConfig holds the information required to interact with metrics objects
+type MetricsConfig struct {
```

Comment:
Given that we have `MetricsConfig` struct, do we need `PrometheusNodeConfig` ?

Reply:
I think that can be done, given they are holding very similar information at this point, but removing it would have many changes which are not in conjunction with this PR.

Reply:
Not sure. The only additional change I think is creating MetricsConfig() object [here](https://github.com/openshift/windows-machine-config-operator/blob/master/controllers/windowsmachine/windowsmachine_controller.go#L89) instead of `PrometheusNodeConfig` .

Reply:
It seems feasible, I was thinking just in terms of renaming changes but that could be another commit as well if we want to do this.